principal_travel/urls.py

```diff
@@ -24,4 +24,8 @@
 urlpatterns = [
     url(r'^admin/', admin.site.urls),
     url(r'^api/', include(router.urls)),
+    url(r'^accounts/', include('registration.backends.simple.urls')), #USE FOR ONE STEP REGISTRATION
+    # url(r'^accounts/', include('registration.backends.default.urls')),
```

Comment:
Including a bit more of the intent here would help the comments be more useful. Something along the lines of "using the simple urls for now; including the two-step registration option below for future use."